prometheus.yml

```
@@ -55,3 +55,7 @@ scrape_configs:
   - job_name: 'kafka-exporter'
     static_configs:
         - targets: [ 'kafka-exporter:9308' ]
+
+  - job_name: 'redis-exporter'
+    static_configs:
+        - targets: [ 'redis-exporter:9121' ]
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 8

(indentation)